src/platform/forms-system/src/js/web-component-patterns/addressPattern.jsx

```diff
@@ -105,6 +105,11 @@ const savedAddress = {
   stateCode: '',
 };
 
+const getAddressPath = (path, index) => {
```

Comment:
address in arrays still doesn't work perfect, but there is a [followup ticket](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team/59225) to address this.